src/commands/heroku.rs

```diff
@@ -39,13 +134,31 @@ fn heroku_client(api_key: &str) -> heroku_rs::client::Heroku {
     Heroku::new(api_key).unwrap()
 }
 
-fn app_response(processed_app_list: Vec<HerokuApp>) -> String {
+fn app_response(app: HerokuApp) -> String {
+    format!(
+        "\nApp ID: {}\nApp Name: {}\nReleased At: {}\nWeb URL: {}\n\n",
+        app.id, app.name, app.released_at, app.web_url
+    )
+}
+
+fn apps_response(processed_app_list: Vec<HerokuApp>) -> String {
     let mut list = String::from("Here are your Heroku apps\n");
 
     for app in processed_app_list {
-        let app_info = format!("App ID: {}\n App Name: {}\n\n", app.id, app.name);
+        let app_info = app_response(app);
         list.push_str(&app_info);
     }
 
     list
 }
+
+fn bot_config(ctx: &Context) -> std::sync::Arc<Config> {
+    let config = ctx
+        .data
+        .read()
+        .get::<Config>()
+        .expect("Expected config")
+        .clone();
+
+    config
```

Comment:
Clippy usually doesn't like a `let` binding immediately followed by returning the value, so you could return the expression without binding it first.

Reply:
Thank you! Forgot to remove that when I was troubleshooting!